docs/new_source.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page
+title: New Data Source
+permalink: /data/new
+nav_order: 1
+has_children: true
+parent: Data sources
+---
+
+# Contributing A New Data Source
+
+Data contributions are welcome. OSV.dev readily accepts security related records from open source projects, given they are provided in the [OSV format](https://ossf.github.io/osv-schema/). Publishing records to OSV is straightforward: reserve a prefix in the OSV-Schema, and then prepare and publish your data via a Git repository or other supported method. The step by step instructions are as follows:
```

Comment:
nit: IMO this paragraph is a bit clunky (though I don't really have any concrete suggestions to improve it).
Maybe there should be a line break before `The step by step instructions are as follows:`